client/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:12.18.3-buster-slim
+
+# Setup variables
+ENV CLIENT_DIRECTORY=/client
+
+# Navigate to client directory
+WORKDIR ${CLIENT_DIRECTORY}
```

Comment:
learning question - what's the point of setting the env here as opposed to just putting the directory in WORKDIR, especially since we put the directory in directly in COPY below?

Reply:
There's no real advantage, it's a vestige of developing these files (there was quite a bit of rearranging while I figured this out). I do however like the consistency in how the directory is defined in both the python and node images.